src/sentry/templates/sentry/emails/activity/generic.html

```diff
@@ -24,7 +24,18 @@
 
     <h2>{{ activity_name }}</h2>
 
-    <p>{{ html_description }}</p>
```

Comment:
When does html_description contain PHI?

Reply:
The `html_description` includes whatever was logged as the error message (from Python, in our case). For debugging this will include dumps of objects or request/response response parameters which often include sensitive information. One example from our application which deals with healthcare data is if a certain function throws an exception it's integral to know the patient ID that caused that exception. That ID is often an MRN which is PHI.